### ISSUE TITLE
Review of french climatic menu

### ISSUE THREAD

**Issue by rachelkg** [africanmathsinitiative/R-Instat#6527]
Hi All, 

I have been having a quick look through the French climatic menu to familiarise myself with how everything should look when I get started on the other menus. 

I have picked up on a few issues which I listed in the attached excel document. I hope it is clear, don't hesitate to let me know if you have any questions of comments. 

[Review of french climatic menu.xlsx](https://github.com/africanmathsinitiative/R-Instat/files/6641320/Review.of.french.climatic.menu.xlsx)

**Comment by rachelkg** [comment]
@lloyddewit Is there a way to see the French dialogues in Visual Studio? I feel like I am having trouble getting the dialogues looking their best when I have to guess at resizing the boxes and keep going back and forth to check. Any insight into how to improve the workflow would be appreciated. Thank you!

**Comment by lloyddewit** [comment]
> 
> 
> @lloyddewit Is there a way to see the French dialogues in Visual Studio? I feel like I am having trouble getting the dialogues looking their best when I have to guess at resizing the boxes and keep going back and forth to check. Any insight into how to improve the workflow would be appreciated. Thank you!

@rachelkg The French text is set at run-time so it's not possible to display the French text while working on the dialogs.
I wonder if there are other ways to avoid having to go back and forth.
Do you use the VisualStudion 'hot reload' feature?
Alternatively could you temporarily change some of the dialog's key text into French while you're updating the dialog box?

I'd be happy to have a quick chat to see if we could come up with any ideas together. I'll send you an email with my contact details.

**Comment by rdstern** [comment]
I note issues as I come across them.  They are all to do with translation, but may not all apply to this pull request:  @rachelkg may be able to advise on some of them as the crowdin translation is improved.
a) `Climatic > File > Import and Tidy Shapefile`  The best translation I have found is for `GIS shape file`, which is simply `fichier SIG`.  I wonder if the English of the menu item should just be `Import GIS shape file` - and omit the tidy, which is quite a recent term in English and not vital to include here.  This would also be done in the label at the top of the dialogue.
b) `Climatic > File > Import from Climsoft` @N-thony edit the sub-dialogue when you press the Establish Connection button.
c) `Climatic > Tidy and Examine > Append` has become Ajouter.  In the Prepare menu it is Superposer les trames de donnees.  That is long, but seems clear to me.  It should be consistent, as it is the same dialogue.  In the dialogue itself we could have the same as the menu item. 
d) `Climatic > Tidy and Examine > Enter daily data`.  In the Options sub-dialogue there are some items that have changed recently and (preumably) need to be added to the crowdin.  They are `Missing/Default Value, Add/Edit New Data Only` and `Include First of the Next Month`.
e)` Climatic > Define Climiatc Data ` The Check Duplicates button needs be bigger.  It is incomplete in French
f) `Climatic > Check Data > Inventory` The Save graphic is longer in French.  Move the control over so, when ticked the label remains complete.  Same for the Missing option - Save the Details the saving control needs to move over.
g) Climatic > Check Data > Display Daily Same as f) for the first 2 options on this dialogue.
h) Climatic > Check Data > Missing values.  This time the saving box on the 1st and 4th button has been moved over too far.  Move it back a bit, to the same place you will use in f) and g).
i) Climatic > Check Data > Missing Values The second option (Add single) move over the bottom control.  In the Methods, with the Constant button the box that now becomes visible needs to become wider for the French.
j) Climatic > Check Data > Boxplot.  Same as usual in f and g.  Move over the saving field.
More to come

.
